SwaglyricsBot/general_commands.py

```diff
@@ -63,8 +63,10 @@ async def send_lyrics():
 
         try:
 
-            await log.add_log("User {} from {} guild requested lyrics".format(
-                ctx.author, ctx.guild if ctx.guild else ctx.channel))
+            await log.add_log(
```

Comment:
this was exceeding 120 chars?

Reply:
I mean, format fixes are fine, but what are the benefits of f-string over formats

Reply:
save space and more readable